in_command.go

```diff
@@ -95,6 +96,18 @@ func (c *InCommand) Run(destDir string, request InRequest) (InResponse, error) {
 				return InResponse{}, err
 			}
 		}
+
+		if foundRelease.PublishedAt != nil || foundRelease.CreatedAt != nil {
+			timestampPath := filepath.Join(destDir, "timestamp")
+			timestamp, err := getTimestamp(foundRelease).MarshalText()
```

Comment:
The date format stored by Concourse is the result of `json.MarshallJson`, which defaults to RFC 3339. Hopefully the default format for `json.MarshalJson` and `json.MarshalText` will never be different